packages/server/database/migrations/20221025161717-addIsFreeMeetingTemplate.ts

```diff
@@ -0,0 +1,18 @@
+import {R} from 'rethinkdb-ts'
+
+const freeTemplateIds = [
+  'startStopContinueTemplate',
+  'gladSadMadTemplate',
+  'whatWentWellTemplate',
+  'workingStuckTemplate',
+  'sailboatTemplate',
+  'original4Template',
+  'fourLsTemplate'
+]
+
+export const up = async function (r: R) {
+  await r.table('MeetingTemplate').getAll(r.args(freeTemplateIds)).update({isFree: true}).run()
+}
+export const down = async function (r: R) {
+  await r.table('MeetingTemplate').getAll(r.args(freeTemplateIds)).update({isFree: false}).run()
```

Comment:
+1 feels like `down()` should clear out the `isFree` properties vs. setting them to false